solara/settings.py

```diff
@@ -36,9 +38,13 @@ class Config:
         env_file = ".env"
 
 
+# in colab or vscode there is not solara cdn proxy available
+_should_not_use_proxy = solara.util.is_running_in_colab() or solara.util.is_running_in_vscode()
+
+
 class Assets(BaseSettings):
     cdn: str = "https://cdn.jsdelivr.net/npm/"
-    proxy: bool = True
+    proxy: bool = not _should_not_use_proxy
```

Comment:
Would this make more sense without the double negative?
```suggestion
    proxy: bool = _should_use_proxy
```